src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -4068,6 +4068,29 @@ def test_check_raw_parameters(self):
         )
         self.assertIsNone(dec_2.check_raw_parameters())
 
+        # custom value
+        dec_2 = AKSPreviewManagedClusterUpdateDecorator(
```

Comment:
this should be dec_3? L4060 uses `dec_2` already. To make these validations easier to read and maintain (why rely on human to count the number?), I suggest we use a table test to list these assertions

Reply:
Ah, I copied the previous test code by mistake which results in reusing the `_2` label. Would remove this piece of code.